meson.build

```diff
@@ -0,0 +1,537 @@
+# /////////////////////////////////////////
+# This file is Licensed MIT, please refer to LICENSE.md for more information.
+# /////////////////////////////////////////
+
+project('voxgui', 'c', 'cpp', version: '0.3.0', license: 'MIT', meson_version: '>=0.62.0', default_options: ['c_std=c11', 'cpp_std=c++11'])
```

Comment:
voxgui??

Reply:
Sorry, messed up the branch a bit, closing this PR, will clean up few things.

Reply:
VoxGUI is a code name project I was working on to create alternative GUI toolkit to GTK and QT that emphasize on ease of binding for other programming languages.